RadixWallet/Core/Resources/Resources/Assets.xcassets/CardCarousel/carousel_icon_continueOnDapp.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "filename" : "carousel_icon_continueOnDapp.png",
```

Comment:
This should be removed, since the dApp icon url will come from Sargon after retrieving metadata.